src/meta/protos/proto/ownership.proto

```diff
@@ -46,10 +46,15 @@ message OwnershipObject {
     string stage = 1;
   }
 
+  message OwnershipWarehouseObject {
+    string uid = 1;
```

Comment:
same as above, what does `u` stands for?

Reply:
unique